### PR TITLE
Migrated from https://github.com/brain-score/brainio_collection.

### PR DESCRIPTION
**Pull request by jjpr-mit**

### CODE REVIEW

**Comment by mschrimpf** [comment]
I think the `scripts/` folder is unnecessary. Let's move everything within that directory up by one level.
Ideally we would also move all the notebooks into their corresponding directories, but it's fine if we don't do that within this PR.

**Comment by jjpr-mit** [comment]
I think it keeps things tidier.  And it's only going to grow over time.

**Comment by mschrimpf** [comment]
let me rephrase: I think `brain-score/packaging/scripts/` is unnecessary, I find `brain-score/packaging/` completely sufficient for what is in there.
